metering/metering-installing-metering.adoc

```diff
@@ -5,5 +5,100 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Installing metering stub
+On {product-title} 4 clusters the Metering Operator can be installed through the OperatorHub. If you are using an earlier version of OpenShift or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use the Metering Operator.
 
+include::modules/metering-install-prerequisites.adoc[leveloffset=+1]
+
+include::modules/metering-install-operator.adoc[leveloffset=+1]
+
+// This content is included directly in the assembly and NOT in a module because the workflow for installing metering requires linking off to the config docs, and xrefs are not allowed in modules. [klamenzo]
+[id="metering-install-metering-stack_{context}"]
+== Installing the Metering stack 
+
+To install the metering stack, create a MeteringConfig resource and add your configuration details. Review
+//xref:metering-configuring-metering.adoc[configuring metering]
+configure metering for more details about all available configuration options. At a minimum you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.
+
+After entering you configuration details, the MeteringConfig resource installs all the necessary metering components inside the openshift-metering namespace.
+
+[IMPORTANT]
+====
+There can only be one MeteringConfig resource in the `openshift-metering` namespace. Any other configuration is not supported.
+====
+
+.Procedure
+
+. From the web console, ensure you are on the Operator Details page for the Metering Operator in the openshift-metering project. You can navigate to this page by clicking *Operators* -> *Installed Operators*, then selecting the Metering Operator.
+
+. Under *Provided APIs*, click *Create Instance* on the MeteringConfig card. This opens a YAML editor with the default configuration file where you can define your configuration.
```

Comment:
I've been testing 4.2 OLM stuff, the card will say "Metering Configuration". So `Under *Provided APIs*, click *Create Instance* on the MeteringConfig card` can be `Under *Provided APIs*, click *Create Instance* on the Metering Configuration card`